### PR TITLE
Warn about missing permissions

### PR DESCRIPTION
**Pull request by chriswahl**
Signed-off-by: Chris Wahl github@wahlnetwork.com

### CODE REVIEW

File: Import-VIRole.ps1 [hunk 1]
```diff
@@ -108,7 +108,16 @@ function Import-VIRole
...
         Write-Verbose -Message 'Parse the permission array for IDs'
-        $PermList = Get-VIPrivilege -Id $PermArray
+        $PermList = Get-VIPrivilege -Id $PermArray -ErrorVariable MissingPerm -ErrorAction SilentlyContinue
+
+        Write-Verbose -Message 'Checking for missing permissions in vCenter'
+        if ($MissingPerm)
+        {
+            foreach ($_ in $MissingPerm)
+            {
+            Write-Warning -Message "Permission named $(($_.Exception.Message.Split("'"))[1]) not found"
```
Comment (rnelson0) [inline Import-VIRole.ps1]: This is the line that should be indented from the `foreach`.

**Comment by chriswahl** [comment]
@rnelson0 Indented :)

**Comment by rnelson0** [comment]
Fixes #3

**Comment by rnelson0** [comment]
Thanks again!

**Pull request merged**